src/PiggyCustomEnchants/Tasks/EffectTask.php

```diff
@@ -65,15 +65,15 @@ public function onRun(int $currentTick)
             if ($enchantment !== null) {
                 $effect = Effect::getEffect(Effect::STRENGTH);
                 $effect->setAmplifier($enchantment->getLevel() - 1);
-                $effect->setDuration(10);
+                $effect->setDuration(8);
```

Comment:
🤦‍♂️

Reply:
10 ticks to 8 ticks... Such a big difference.

Reply:
try pvping ull see :P

Reply:
10 ticks is literally half a second...